gprotorch/kernels/gnn_kernels/test_hu_iclr_2020.py

```diff
@@ -0,0 +1,357 @@
+"""
+Reformatted and cleaned up GNN modules from Hu et al.
+Strategies for Pre-training Graph Neural Networks. ICLR 2020
+(https://github.com/snap-stanford/pretrain-gnns)
```

Comment:
Should we also give the module this code is based on i.e. model.py in the linked repo?

Reply:
Good point.